### PR TITLE
BRITECHSRM-399: Speaker patch method

### PR DESCRIPTION
**Pull request by jebbinSCL**

### CODE REVIEW

File: ApiGateway/ProfilesRepository.fs [hunk 1]
```diff
@@ -80,7 +80,25 @@ let updateProfile (pid : Guid) (profile : ProfileDto) =
         match result.StatusCode with
         | HttpStatusCode.OK -> getProfile pid
         | _ -> 
-            Log.Information("Error updating profile Status code: {statusCode}. Reason: {reasonPhrase}", result.StatusCode, result.ReasonPhrase)
+            Log.Information("Error updating profile. Status code: {statusCode}. Reason: {reasonPhrase}", result.StatusCode, result.ReasonPhrase)
+            Failure { HttpStatusCode = result.StatusCode; Body = result.ReasonPhrase }
+    with
+        | ex ->
```
Comment (pallenSL) [inline ApiGateway/ProfilesRepository.fs]: Should the indentation match "match" ?  i.e. one less tab here.

Comment (jebbinSCL) [inline ApiGateway/ProfilesRepository.fs]: Which line are you referring to? Line 86 is inline with the match.

Comment (pallenSL) [inline ApiGateway/ProfilesRepository.fs]: I mean the | inline with the "with", as the | is inline with the "match".

Comment (jebbinSCL) [inline ApiGateway/ProfilesRepository.fs]: Looks like this is actually correct, there should be a tab after the with. I'll adjust other places accordingly in this file, and do the rest as I make changes.

Comment (jebbinSCL) [inline ApiGateway/ProfilesRepository.fs]: Actually all the tabs after with are in the right place in this file.

**Comment by pallenSL** [comment]
The code structure is much clearer (to me) this way.
:shipit:

**Pull request closed without merging**